.openapidoc/config.js

```diff
@@ -73,8 +73,9 @@ function calculateSpecVersion(specFile) {
 }
 
 function isReleaseVersion(specVersion) {
-  // our main project's gradle's build calculateVersion puts -dev- for snapshot version
-  return !specVersion.includes("-dev-");
+  // our main project's gradle's build calculateVersion adds "+<new commits since stable>-<hash>"
+  // after the version for dev builds
+  return !specVersion.includes("+");
```

Comment:
suggestion: `return !specVersion.includes("+") && !specVersion.includes("-dev-");`

This will allow to manage both both versioning while switch happens.

Reply:
nvm ... the main PR is already in.